test/Microsoft.OpenApi.Readers.Tests/V2Tests/Samples/OpenApiParameter/headerParameter.yaml

```diff
@@ -0,0 +1,10 @@
+﻿# https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameterObject
```

Comment:
Do we include this line?

Reply:
I think we should point to the source we took the YAML files from.